### PR TITLE
Comments on manucript

### PR DESCRIPTION
**Pull request by TCLamnidis**

### CODE REVIEW

File: content/03.results-and-discussion.md [hunk 1]
```diff
@@ -15,33 +15,33 @@ nf-core/eager provides improved efficiency over the original EAGER pipeline by r
...
-   3. Preprocessing statistics generation
+   a. Sequencing quality control: FastQC (@url:https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
+   b. Sequencing artefact clean-up: AdapterRemoval2 (@doi:10.1186/s13104-016-1900-2)
+   c. Preprocessing statistics generation
 2. Mapping and post- processing
-   1. Alignment against reference genome: BWA (@doi:10.1093/bioinformatics/btp324, @doi:10.1093/bioinformatics/btp698, @arXiv:1303.3997), CircularMapper (@doi:10.1186/s13059-016-0918-z)
-   2. Mapping quality filtering: Samtools (@doi:10.1093/bioinformatics/btp352)
-   3. PCR duplicate removal: DeDup (@doi:10.1186/s13059-016-0918-z), Picard MarkDuplicates(@doi:10.1101/gr.107524.110)
-   4. Mapping statistics generation: PreSeq (@doi:10.1038/nmeth.2375), Qualimap2(@doi:10.1093/bioinformatics/btv566)
+   a. Alignment against reference genome: BWA (@doi:10.1093/bioinformatics/btp324, @doi:10.1093/bioinformatics/btp698, @arXiv:1303.3997), CircularMapper (@doi:10.1186/s13059-016-0918-z)
```
Comment (jfy133) [inline content/03.results-and-discussion.md]: Nope, this breaks markdown, please revert.

File: content/03.results-and-discussion.md [hunk 2]
```diff
@@ -15,33 +15,33 @@ nf-core/eager provides improved efficiency over the original EAGER pipeline by r
...
 
-We have further extended functionality of the pipeline, primarily focusing on ancient metagenomic analysis to be run alongside the standard genomic analysis against a single reference genome. We have added the ability to screen all off-target reads (from mapping to the supplied reference FASTA) with two metagenomic profilers: MALT (@doi:10.1101/050559, @doi:10.1038/s41559-017-0446-6) and Kraken2 (@doi:10.1186/s13059-019-1891-0). Characterisation of properties of authentic ancient DNA (@doi:10.1073/pnas.0704665104) from MALT alignments is carried out with the HOPS pipeline (@doi:10.1186/s13059-019-1903-0). Ancient metagenomic studies sometimes may include comparative samples from living day individuals (@doi:10.1186/s40168-019-0717-3). To support open data, whilst respecting data privacy, nf-core/eager includes a 'strip_fastq' script (by M.B.) which creates raw FASTQ files but with reference-genome mapped reads removed. This then allows safe upload of sequencing data to public repositories with identifiable data removed.
```
Comment (jfy133) [inline content/03.results-and-discussion.md:35]: Why do you want to remove the initials?

File: content/03.results-and-discussion.md [hunk 3]
```diff
@@ -15,33 +15,33 @@ nf-core/eager provides improved efficiency over the original EAGER pipeline by r
...
+We have further extended functionality of the pipeline, primarily focusing on ancient metagenomic analysis to be run alongside the standard genomic analysis against a single reference genome. We have added the ability to screen all off-target reads (from mapping to the supplied reference FASTA) with two metagenomic profilers: MALT (@doi:10.1101/050559, @doi:10.1038/s41559-017-0446-6) and Kraken2 (@doi:10.1186/s13059-019-1891-0). Characterisation of properties of authentic ancient DNA (@doi:10.1073/pnas.0704665104) from MALT alignments is carried out with the HOPS pipeline (@doi:10.1186/s13059-019-1903-0). Ancient metagenomic studies sometimes may include comparative samples from present-day individuals (@doi:10.1186/s40168-019-0717-3). To support open data, whilst respecting data privacy, nf-core/eager includes a 'strip_fastq' script which creates raw FASTQ files but with reference-genome mapped reads removed. This then allows safe upload of sequencing data to public repositories with identifiable human data removed.
```
Comment (jfy133) [inline content/03.results-and-discussion.md]:
```suggestion
We have further extended functionality of the pipeline, primarily focusing on ancient metagenomic analysis to be run alongside the standard genomic analysis against a single reference genome. We have added the ability to screen all off-target reads (from mapping to the supplied reference FASTA) with two metagenomic profilers: MALT (@doi:10.1101/050559, @doi:10.1038/s41559-017-0446-6) and Kraken2 (@doi:10.1186/s13059-019-1891-0). Characterisation of properties of authentic ancient DNA (@doi:10.1073/pnas.0704665104) from MALT alignments is carried out with the HOPS pipeline (@doi:10.1186/s13059-019-1903-0). Ancient metagenomic studies sometimes may include comparative samples from living individuals (@doi:10.1186/s40168-019-0717-3). To support open data, whilst respecting data privacy, nf-core/eager includes a 'strip_fastq' script which creates raw FASTQ files but with reference-genome mapped reads removed. This then allows safe upload of sequencing data to public repositories with identifiable human data removed.
```

File: content/03.results-and-discussion.md [hunk 4]
```diff
@@ -15,33 +15,33 @@ nf-core/eager provides improved efficiency over the original EAGER pipeline by r
...
+Alongside the portable new pipeline report, we have written extensive documentation on all parts of running and interpreting the output of the pipeline. Given that a large fraction of aDNA researchers come from fields outside computational biology, and thus may have limited computational training, we have written documentation that also gives guidance on how to interpret each section of the report, specifically in the context of NGS and aDNA. This includes schematic images of best practices or expected output that published under CC-BY licenses to allow for use in other training material. We hope this open-access resource will make the aDNA discipline more accessible to researchers new to the field, by providing practical and 'applied' knowledge as to how aDNA characteristics translate to downstream analyses.
```
Comment (jfy133) [inline content/03.results-and-discussion.md:45]: I was trying to emphasis many aDNA researchers is not from any super-technical discpline (chemists or engineers will often have some comp. training) who get 0% computational training except for word and excel. Can you suggest a rephrase?

**Review by jfy133 — CHANGES_REQUESTED** [review]

File: content/03.results-and-discussion.md (hunk 2 above)
Comment (TCLamnidis) [reply content/03.results-and-discussion.md:35]: Is that the normal way to do this in bioinfo papers? seems strange to me that the specific scripts are attributed to collaborators in the main text. That is what the "Author contributions" section is for in papers.

File: content/03.results-and-discussion.md (hunk 4 above)
Comment (TCLamnidis) [reply content/03.results-and-discussion.md:45]: Isn't my existing suggestions covering that?
> "Given that a large fraction of aDNA researchers come from fields outside computational biology, **and thus may have limited computational training**, we have written documentation [...]"

File: content/03.results-and-discussion.md (hunk 1 above)
Comment (TCLamnidis) [reply content/03.results-and-discussion.md]: OOF, my bad! Converted to bullet points so there aren't nested numbers.

File: content/03.results-and-discussion.md (hunk 2 above)
Comment (apeltzer) [reply content/03.results-and-discussion.md:35]: Yeah normally thats not used - either make people co-author (which all of them are) or have them for very minor stuff in acknowledgments ;-)

**Review by apeltzer — APPROVED** [review]

File: content/03.results-and-discussion.md (hunk 2 above)
Comment (jfy133) [reply content/03.results-and-discussion.md:35]: Ok fair enough. Was trying to give more direct attribution to 'unpublished' tools, but I guess not necessary. Author contributions don't normally go into that much detail.

File: content/03.results-and-discussion.md (hunk 4 above)
Comment (jfy133) [reply content/03.results-and-discussion.md:45]: Fair enough. Maybe I'm going into too much 'PR' mode and selling it too hard...

**Review by jfy133 — APPROVED** [review]

**Pull request merged**